openedx_external_enrollments/factory.py

```diff
@@ -0,0 +1,19 @@
+"""Openedx external enrollments factory file."""
+from openedx_external_enrollments.external_enrollments import (
+    EdxEnterpriseExternalEnrollment,
+    EdxInstanceExternalEnrollment,
+)
+
+
+class ExternalEnrollmentFactory:
+    """
+    """
+    @classmethod
+    def get_enrollment_controller(cls, controller):
+        """
+        Return the an instance of the enrollment controller.
+        """
+        if controller.lower() == 'openedx':
+            return EdxInstanceExternalEnrollment()
+        else:
```

Comment:
Unnecessary else statement. @andrey-canon

Reply:
out of the scope of this pr